lib/typescript/rules-typescript.js

```diff
@@ -0,0 +1,33 @@
+// TYPESCRIPT ESLINT CONFIG
+module.exports = {
+  rules: {},
+  overrides: [
```

Comment:
IIRC these rules are in the overrides section so that they're only disabled for ts/tsx files. Having them here means that, for example no-unused-vars will still work for .js files.

It does mean that to disable/override these rules you'd need to use the overrides section of the eslint file rather than the rules section (thanks pablo)

Reply:
added comments here about what is happening